<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary-Deciminal Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="txtNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="binaryButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decimalButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Deciminal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>github.com/AHXR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>279</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBinary_Deciminal_Converter">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuBits">
     <property name="title">
      <string>Bits</string>
     </property>
     <addaction name="action8_2"/>
     <addaction name="action16"/>
     <addaction name="action32"/>
     <addaction name="action64"/>
     <addaction name="action128"/>
     <addaction name="action256"/>
    </widget>
    <addaction name="menuBits"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuBinary_Deciminal_Converter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action8_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action32">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>32</string>
   </property>
  </action>
  <action name="action16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="action64">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>64</string>
   </property>
  </action>
  <action name="action128">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>128</string>
   </property>
  </action>
  <action name="action256">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>256</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
